### PR TITLE
normalizes code, style and strict mode mariadb

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 1]
```diff
@@ -1,543 +1,694 @@
...
+                $val ? $rq = "INSERT INTO `mod_dsm_pool` VALUES (".$val.")" : $rq = null;
+                $dbResult = $pearDB->query($rq);
+                $dbResult = $pearDB->query("SELECT MAX(pool_id) FROM `mod_dsm_pool`");
+                $cmd_id = $dbResult->fetch();
+            }
+        }
+    }
+}
+
+    /**
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 2]
```diff
@@ -1,543 +1,694 @@
...
+    if (!isset($oldPrefix)) {
+        $oldPrefix = "213343434334343434343";
+    }
+
+    $dbResult = $pearDB->query(
+        "SELECT service_id, service_description " .
+        "FROM service s, host_service_relation hsr " .
+        "WHERE hsr.host_host_id = '$host_id' " .
+        "AND service_id = service_service_id " .
+        "AND service_description LIKE '$oldPrefix%' ORDER BY service_description ASC"
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]:
```suggestion
        "AND service_description LIKE ' . $oldPrefix% . ' ORDER BY service_description ASC"
```

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 3]
```diff
@@ -1,543 +1,694 @@
...
+    isset($ret["pool_activate"]["pool_activate"])
+        && $ret["pool_activate"]["pool_activate"] != null
+        ? $rq .= "'" . $ret["pool_activate"]["pool_activate"] . "', ": $rq .= "NULL, ";
+    isset($ret["pool_service_template_id"])
+        && $ret["pool_service_template_id"] != null
+        ? $rq .= "'" . $ret["pool_service_template_id"] . "' ": $rq .= "NULL ";
+    $rq .= ")";
+
+    /*
+        * Generate all services
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 4]
```diff
@@ -1,543 +1,694 @@
...
+        */
+    $dbResult = $pearDB->query("SELECT pool_prefix FROM mod_dsm_pool WHERE pool_id = '$pool_id'");
+    $data = $dbResult->fetch();
+    $oldPrefix = $data["pool_prefix"];
+
+    $ret = array();
+    $ret = $form->getSubmitValues();
+
+    /*
+        * Validate if host is not already use
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 5]
```diff
@@ -1,543 +1,694 @@
...
+function updatePool($pool_id = null)
+{
+    global $form, $pearDB;
+
+    if (!$pool_id) {
+        return false;
+    }
+
+    /*
+        * Get Old Prefix
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.php [hunk 6]
```diff
@@ -1,238 +1,261 @@
...
+if ($o == "a") {
+    $form->addElement('header', 'title', _("Add a pool of services"));
+} elseif ($o == "c") {
+    $form->addElement('header', 'title', _("Modify a pool of services"));
+} elseif ($o == "w") {
+    $form->addElement('header', 'title', _("View a pool of services"));
+}
+
+/*
+    * pool basic information
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.php [hunk 7]
```diff
@@ -1,238 +1,261 @@
...
+    */
+$form->addElement('header', 'information', _("General Information"));
+$form->addElement('header', 'slotInformation', _("Slots Information"));
+$form->addElement('header', 'Notification', _("Notifications Information"));
+
+
+/*
+    * No possibility to change name and alias, because there's no interest
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.php [hunk 8]
```diff
@@ -1,238 +1,261 @@
...
+$form->addElement('text', 'pool_description', _("Description"), $attrsText);
+$form->addElement('text', 'pool_number', _("Number of Slots"), $attrsTextSmall);
+$form->addElement('text', 'pool_prefix', _("Slot name prefix"), $attrsText);
+$form->addElement('select', 'pool_host_id', _("Host Name"), $poolHost);
+$form->addElement('select', 'pool_cmd_id', _("Check commands"), $Cmds);
+$form->addElement('text', 'pool_args', _("arguments"), $attrsText2);
+$form->addElement('select', 'pool_service_template_id', _("Service template based"), $poolST);
+
+/*
+    * Further informations
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.php [hunk 9]
```diff
@@ -1,238 +1,261 @@
...
+        $from_list_menu = false;
+    } else {
+        $from_list_menu = true;
+    }
+}
+$form->setRequiredNote("<font style='color: red;'>*</font>&nbsp;" . _("Required fields"));
+
+
+/*
+    * Smarty template Init
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.php [hunk 10]
```diff
@@ -1,238 +1,261 @@
...
+        )
+    );
+    $form->freeze();
+}
+$action = $form->getSubmitValue("action");
+if ($valid && $action["action"]["action"]) {
+    include $path . "listSlot.php";
+} else {
+    /*
+        * Apply a template definition
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/slots.php [hunk 11]
```diff
@@ -1,77 +1,102 @@
...
+isset($_POST["select"]) ? $cP = $_POST["select"] : $cP = null;
+$cG ? $select = $cG : $select = $cP;
+
+isset($_GET["dupNbr"]) ? $cG = $_GET["dupNbr"] : $cG = null;
+isset($_POST["dupNbr"]) ? $cP = $_POST["dupNbr"] : $cP = null;
+$cG ? $dupNbr = $cG : $dupNbr = $cP;
+
+$search = isset($_POST['searchSlot']) ? htmlentities($_POST['searchSlot'], ENT_QUOTES) : null;
+
+    /*
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/slots.php]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/slots.php [hunk 12]
```diff
@@ -1,77 +1,102 @@
...
+    * Path to the configuration dir
+    */
+$path = "./modules/centreon-dsm/core/configuration/services/";
+
+/*
+    * PHP functions
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/slots.php]: wrong indentation

**Comment by lgcosta** [comment]
Hi @sc979 , please review changes

File: www/modules/centreon-dsm/core/configuration/services/formSlot.ihtml [hunk 13]
```diff
@@ -3,40 +3,40 @@
...
-	{$form.hidden}
+    <div id="validFormTop">
+    {if $o == "a" || $o == "c" || $o == "mc"}
+        <p class="oreonbutton">{$form.submitC.html}{$form.submitMC.html}{$form.submitA.html}&nbsp;&nbsp;&nbsp;{$form.reset.html}</p>
+    {else if $o == "w"}
+        <p class="oreonbutton">{$form.change.html}</p>
+    {/if}
+    </div>
+    <div id='tab1' class='tab'>
+     <table class="formTable table">
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.ihtml]: wrong indentation

File: www/modules/centreon-dsm/core/configuration/services/formSlot.ihtml [hunk 14]
```diff
@@ -3,40 +3,40 @@
...
+        <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="pool_number" /><span>{$form.pool_number.label}</span></td><td class="FormRowValue">{$form.pool_number.html}</td></tr>
+        <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="pool_prefix" /><span>{$form.pool_prefix.label}</span></td><td class="FormRowValue">{$form.pool_prefix.html}</td></tr>        
+        <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="pool_cmd_id" /><span>{$form.pool_cmd_id.label}</span></td><td class="FormRowValue">{$form.pool_cmd_id.html}&nbsp;{$form.pool_args.html}</td></tr>         
+        <tr class="list_lvl_1"><td class="ListColLvl1_name" colspan="2"><h4>{$form.header.furtherInfos}</h4></td></tr>
+        <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="pool_activate" /><span>{$form.pool_activate.label}</span></td><td class="FormRowValue">{$form.pool_activate.html}</td></tr>
+        {if $o == "a" || $o == "c"}
+            <tr class="list_lvl_2"><td class="ListColLvl2_name" colspan="2">{$form.required._note}</td></tr>
+        {/if}
+    </table>
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/formSlot.ihtml]: please check line 14 comment

File: www/modules/centreon-dsm/core/configuration/services/listSlot.ihtml [hunk 15]
```diff
@@ -1,67 +1,67 @@
...
+        <tr>
+            <th><h5>{t}Filters{/t}</h5></th>
+        </tr>
+        <tr>
+            <td><h4>{$searchLabel}</h4></td>
+        </tr>
+        <tr>
+            <td><input type='text' name='searchSlot' value='{$search}' /></td>
+            <td><input type='submit' name='SearchButton' value='{$searchLabel}' class="btc bt_success" /></td>
+    </tr>
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/listSlot.ihtml]: wrong indentation. same for next line

File: www/modules/centreon-dsm/core/configuration/services/listSlot.ihtml [hunk 16]
```diff
@@ -1,67 +1,67 @@
...
+        <tr class="ToolbarTR">
+            <td>
+                {$msg.options} {$form.o2.html}
+                &nbsp;&nbsp;&nbsp;
+                {if $ldap == "1" }<a href="{$msg.ldap_importL}" class="btc bt_success">{$msg.ldap_importT}</a>&nbsp;-&nbsp;{/if}
+                <a href="{$msg.addL}" class="btc bt_success">{$msg.addT}</a>
+                <input name="p" value="{$p}" type="hidden">
+            </td>            
+            {php}
+               include('./include/common/pagination.php');
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/listSlot.ihtml]: comment -> this php file, shouldn't be imported here. Please

Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/listSlot.ihtml]:
```suggestion
               include_once('./include/common/pagination.php');
```

File: www/modules/centreon-dsm/core/configuration/services/DB-Func.php [hunk 17]
```diff
@@ -361,10 +361,10 @@ function generateServices($prefix, $number, $host_id, $template, $cmd, $args, $o
                 $pearDB->query(
                     "INSERT INTO host_service_relation (
                         service_service_id, host_host_id
-                    ) VALUES ('$service_id', '" . $host_id . "')"
+                    ) VALUES ('" . $service_id . "', '" . $host_id . "')"
                 );
                 $pearDB->query(
-                    "INSERT INTO extended_service_information (service_service_id) VALUE ('$service_id')"
+                    "INSERT INTO extended_service_information (service_service_id) VALUES ('" . $service_id . "')"
```
Comment (sc979) [inline www/modules/centreon-dsm/core/configuration/services/DB-Func.php]: comment -> glad you see it. One less bug ;)
same comment for line 450

**Comment by sc979** [comment]
Hi @lgcosta ,
New files, new requests ;)
Thanks for your work,
Regards,
sc979

**Comment by lgcosta** [comment]
done @sc979 

Thanks !

**Comment by lgcosta** [comment]
hey @sc979 or @loiclau can you merge this?

**Pull request merged**